tools/db_ping.rb

```diff
@@ -1,6 +1,6 @@
 total = 0
 5.times do
-  ping = EvmDatabase.ping
+  ping = EvmDatabase.ping(ApplicationRecord.connection)
```

Comment:
Can this just be defaulted in EvmDatabase.ping?  Seems strange to have to pass one.